sunraster/instr/spice.py

```diff
@@ -210,18 +202,18 @@ def _read_single_spice_l2_fits(
                 # Define WCS from new header
                 wcs = WCS(new_header)
                 # Define exposure times from metadata.
-                exp_times = TimeDelta(np.repeat(meta.get("XPOSURE"), hdu.data.shape[-1]), format="sec")
+                exp_times = u.Quantity(np.zeros(hdu.data.shape[-1]) + meta.get("XPOSURE"), unit=u.s)
                 # Define data cube.
                 data = hdu.data
                 spectrogram = SpectrogramCube(
                     data=data,
                     wcs=wcs,
                     mask=np.isnan(data),
                     unit=u.adu,
-                    extra_coords=[("exposure time", -1, exp_times)],
                     meta=meta,
                     instrument_axes=("raster scan", "spectral", "slit", "slit step"),
                 )
+                spectrogram.meta.add("exposure time", exp_times, None, 0)
```

Comment:
```suggestion
                spectrogram.meta.add("exposure time", exp_times, None, 3)
```
My previous change meant exposure time pointing to the wrong axis.  (I mixed up SPICE and IRIS data.)  This change should fix the failing tests.